Assets/Scripts/UI/StatBar.cs

```diff
@@ -0,0 +1,93 @@
+﻿using System;
+using System.Collections;
+using System.Collections.Generic;
+using UnityEditor;
+using UnityEngine;
+using UnityEngine.UI;
+
+public class StatBar : MonoBehaviour
+{
+    public string SpriteLocation = "Sprites/PlayerHungerStates_Sprite";
+    
+    private int _statusValue = 100;
+
+    private Sprite[] _barStates;
```

Comment:
This can be a public field → Sprites can be dragged into the array in the inspector, so the sprite location is not needed

Reply:
serialized, see further up